src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -1223,6 +1225,7 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
             pageIndex={page}
             pageSize={perPage}
             items={savedObjects}
+            workspaceIdNameMap={workspaceIdNameMap}
```

Comment:
I think you can do this instead of having another state
```suggestion
            workspaceIdNameMap={this.workspaceNameIdLookup}
```